Update/tsuk_009.txt

```diff
@@ -1759,7 +1759,7 @@ void main()
 //r梨花ちゃんが何かを知ってしまっているのなら、早急に手を打たねば。はやる気持ちでそう考えていた俺は、ちょっと拍子抜けしてしまった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "梨花ちゃんが何かを知ってしまっているのなら、早急に手を打たねば。はやる気持ちでそう考えていた俺は、ちょっと拍子抜けしてしまった。",
-			NULL, "If Rika-chan knew something, we needed to deal with it right away. But I was a little disappointed that my impulse thought went there.", Line_Normal);
+			NULL, "Since my mind had been entired geared on doing something if Rika-chan knew anything, Rena's response threw me off a little.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Since my mind had been entirely geared on doing something if Rika-chan knew anything, Rena's response threw me off a little.", Line_Normal);
```